bogota-abril-2019/css-magazine-cover/laura_bruges-task2-magazine/index.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <title>NatGeo black and white</title>
+    <link href="https://fonts.googleapis.com/css?family=Old+Standard+TT:400,700|Work+Sans:500" rel="stylesheet"> 
+    <link rel="stylesheet" type="text/css" href="styles/magazine.css"/>
+</head>
+<body>
+    <article id="margin">
+        <aside id="girls-container" class="container"></aside>
+        <section id="main-container" class="container">
+            <header id="headlines">

```

Comment:
Usally: 
```
The <header> element represents a container for introductory content or a set of navigational links.
/w3 schools
```
So we should have 3 in the same article. Seems odd.

Reply:
Fixed 😄